crashreports/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(TARGET crashreports)
+set(TARGET_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+
+message( STATUS "========== Configuring ${TARGET} ==========" )
+
+add_library(${TARGET} STATIC)
+
+set(SOURCES "")
+set(INCLUDES INTERFACE ./)
+set(LIBRARIES "")
+set(DEFINIES "")
+
+add_subdirectory(crashreporter)
+
+list(APPEND SOURCES
+    PRIVATE
+        BreakpadConfigurer.h
+        BreakpadConfigurer.cpp
+)
+list(APPEND LIBRARIES
+    PRIVATE
+        breakpad
+)
+list(APPEND DEFINES
+    PUBLIC
+        -DUSE_BREAKPAD
+)
+
+if(WIN32)
+    list(APPEND SOURCES
+        PRIVATE
+            internal/win32/CrashReportContext.h
+            internal/win32/CrashReportContext.cpp
+    )
+elseif(UNIX)
+    list(APPEND SOURCES
+        PRIVATE
+            internal/unix/CrashReportContext.h
```

Comment:
Thank you for including all the relevant files in SOURCES, even the .h files.  Which are not needed to make it build, but add convenience in the IDE browser.